storage_service/locations/api/resources.py

```diff
@@ -465,6 +491,34 @@ def dehydrate_misc_attributes(self, bundle):
         # Serialize JSONField as dict, not as repr of a dict
         return bundle.obj.misc_attributes
 
+    def hydrate_current_location(self, bundle):
```

Comment:
Here we teach tastypie how to convert `current_location` into a real location when the value matches the regular expression `default_location_regex`, e.g. if `current_location=/api/v2/location/default/AS/`, this method will convert the value into `/api/v2/location/<uuid>/`.